eksctl/nasa-ghg.jsonnet

```diff
@@ -27,7 +27,7 @@ local nodeAz = "us-west-2a";
 local notebookNodes = [
     { instanceType: "r5.xlarge" },
     { instanceType: "r5.4xlarge" },
-    { instanceType: "r5.16xlarge" },
+    { instanceType: "r5.16xlarge" }
```

Comment:
```suggestion
    { instanceType: "r5.16xlarge" },
```

Reply:
@batpad i don't have permissions to accept this change on this PR apparently, can you?